pandas/core/apply.py

```diff
@@ -1,15 +1,16 @@
 import warnings
+
 import numpy as np
-from pandas import compat
+
 from pandas._libs import reduction
-from pandas.core.dtypes.generic import ABCSeries
-from pandas.core.dtypes.common import (
-    is_extension_type,
-    is_dict_like,
-    is_list_like,
-    is_sequence)
 from pandas.util._decorators import cache_readonly
 
+from pandas.core.dtypes.common import (
+    is_dict_like, is_extension_type, is_list_like, is_sequence)
+from pandas.core.dtypes.generic import ABCSeries
+
+from pandas import compat
```

Comment:
Can you make this “import pandas.compat as compat”?  That will group it with the appropriate level of dependencies.  Same in other files if it comes up.

Reply:
Thanks for this ~11 files in this PR. ( Will make this change in my other PR too )